index.html

```diff
@@ -48,8 +50,252 @@ <h1 class="main-name">Erik Stoupignan</h1>
         <div class="prueba">
             <img src="/assets/image_geometry_2.png" alt="Decorative photo" class="background-img2">
         </div>
-
     </main>
+
+    <!-- Proyects section -->
+    <section class="work-project">
+        <h1 class="main-name">Projects</h1>
+
+        <div class="project">
+            <ul>
+                <li>
+                    <!-- Section  #1-->
+                    <div class="container">
+                        <img class="box1" src="/assets/Rectangle21.png" alt="Decoration photo">
+                        <div class="box box2">
+                            <h2 class="subtitle">Project names goes here</h2>
+                        </div>
+                        <!-- Button's Code -->
+                        <div class="box align box3">
+                            <p class="code-button">HTML/CSS</p>
+                        </div>
+                        <div class="box align box4">
+                            <p class="code-button">Ruby on Rails</p>
+                        </div>
+                        <div class="box align box5">
+                            <p class="code-button">JavaScript</p>
+                        </div>
```

Comment:
Hi @ErikStoupignan , what your previous reviewer meant was to wrap this section (technologies used) in a list tag and make each technology a list item, instead, you made the whole work section a list and each card a list item. Kindly fix it.

Reply:
@muneebulrehman Hello, Thanks for explaining it to me. Honestly, I had not understood but thanks to your comments, I was able to fix it.